Cannot build with node v6 or v8

Hi,
thank you first for your work! I was about to test your module, but it seems I can only build it with nodejs v0.12.x...

with v6.11.1 I have this output:
```sh
In file included from ../src/dbus.cc:7:0:
../../nan/nan.h: At global scope:
../../nan/nan.h:602:20: error: variable or field ‘AddGCEpilogueCallback’ declared void
       v8::Isolate::GCEpilogueCallback callback
                    ^~~~~~~~~~~~~~~~~~
../../nan/nan.h:602:7: error: ‘GCEpilogueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCEpilogueCallback callback
       ^~
../../nan/nan.h:603:18: error: expected primary-expression before ‘gc_type_filter’
     , v8::GCType gc_type_filter = v8::kGCTypeAll) {
                  ^~~~~~~~~~~~~~
../../nan/nan.h:608:20: error: variable or field ‘RemoveGCEpilogueCallback’ declared void
       v8::Isolate::GCEpilogueCallback callback) {
                    ^~~~~~~~~~~~~~~~~~
../../nan/nan.h:608:7: error: ‘GCEpilogueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCEpilogueCallback callback) {
       ^~
../../nan/nan.h:613:20: error: variable or field ‘AddGCPrologueCallback’ declared void
       v8::Isolate::GCPrologueCallback callback
                    ^~~~~~~~~~~~~~~~~~
../../nan/nan.h:613:7: error: ‘GCPrologueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCPrologueCallback callback
       ^~
../../nan/nan.h:614:18: error: expected primary-expression before ‘gc_type_filter’
     , v8::GCType gc_type_filter = v8::kGCTypeAll) {
                  ^~~~~~~~~~~~~~
../../nan/nan.h:619:20: error: variable or field ‘RemoveGCPrologueCallback’ declared void
       v8::Isolate::GCPrologueCallback callback) {
                    ^~~~~~~~~~~~~~~~~~
../../nan/nan.h:619:7: error: ‘GCPrologueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCPrologueCallback callback) {
       ^~
../../nan/nan.h: In function ‘bool Nan::SetAccessor(v8::Local<v8::Object>, v8::Local<v8::String>, Nan::GetterCallback, Nan::SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)’:
../../nan/nan.h:1961:16: warning: ‘bool v8::Object::SetAccessor(v8::Local<v8::Name>, v8::AccessorNameGetterCallback, v8::AccessorNameSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
     , attribute);
                ^
In file included from /home/user/.node-gyp/6.11.1/include/node/v8.h:25:0,
                 from ../src/dbus.cc:1:
/home/user/.node-gyp/6.11.1/include/node/v8.h:2751:22: note: declared here
                 bool SetAccessor(Local<Name> name,
                      ^
/home/user/.node-gyp/6.11.1/include/node/v8config.h:333:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
dbus.target.mk:115: recipe for target 'Release/obj.target/dbus/src/dbus.o' failed
make: *** [Release/obj.target/dbus/src/dbus.o] Error 1
make: Leaving directory '/home/user/tmp/dbus-test/node_modules/dbus2/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/user/.nvm/versions/node/v6.11.1/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Linux 4.10.0-35-generic
gyp ERR! command "/home/user/.nvm/versions/node/v6.11.1/bin/node" "/home/user/.nvm/versions/node/v6.11.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /home/user/tmp/dbus-test/node_modules/dbus2
gyp ERR! node -v v6.11.1
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 
npm WARN dbus-test@1.0.0 No description
npm WARN dbus-test@1.0.0 No repository field.
npm ERR! Linux 4.10.0-35-generic
npm ERR! argv "/home/user/.nvm/versions/node/v6.11.1/bin/node" "/home/user/.nvm/versions/node/v6.11.1/bin/npm" "install" "dbus2" "--save"
npm ERR! node v6.11.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! dbus2@1.0.1 install: `node-gyp configure build`
npm ERR! Exit status 1
```

Any clue on this?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "dbus2",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "dbus binding for Node.js",
   "author": "Shouqun Liu <liushouqun@gmail.com>",
   "contributors": [
@@ -19,7 +19,7 @@
     "test": "tap test/**/*.test.js"
   },
   "dependencies": {
-    "nan": "2.1.0"
+    "nan": "^2.1.0"
   },
   "os": [
     "!win32"

